### PR TITLE
SPCTR-3123: Improvement - Added vertical alignment for Testimonial content on equal height

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Providing vertical alignment options when equal height of the testimonial block is enabled.
https://brainstormforce.atlassian.net/browse/SPCTR-3123

### Screenshots

https://d.pr/v/IYSEPy

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Improvement: Testimonial - Added vertical alignment option for content on equal height.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Improvement: Testimonial - Added a vertical alignment option for equal-height content.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update needed.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

**Review by bsf-zanev — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

File: src/blocks/testimonial/settings.js [hunk 2]
```diff
@@ -1193,6 +1194,32 @@ const Settings = ( props ) => {
...
+								label={ __( 'Vertical Alignment', 'ultimate-addons-for-gutenberg' ) }
+								data={ {
+									value: vAlignContent,
+									label: 'vAlignContent',
+								} }
+								className="uagb-multi-button-alignment-control"
+								options={ [
+									{
+										value: 'flex-start',
+										label: 'Top',
```
Comment (adi3890) [inline src/blocks/testimonial/settings.js:1209]: Should this be translatable. Please confirm for other labels too.

Comment (rajmohanreddy774) [reply src/blocks/testimonial/settings.js]: Yes, Added.

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**